PRINCIPLES.md

```diff
@@ -3,6 +3,28 @@ These are principles we try to follow as a team working on this product. Anyone
 ## 0. Application architecture
 1. Use React as a thin view layer unless absolutely necessary, presenting state that is defined outside of React.
 
+2. Reduce the usage of the global `initPromise()` workflow of globally importing and initializing the WASM instance.
+
+3. Reduce the usage of circular dependencies in src/lib/singletons.ts.
```

Comment:
Terms like "reduce" make me think these shouldn't be listed as principles, but rather burned down as issues like #8885 or #6032 and then maintained through principles like 4 here below.

Reply:
I guess in my opinion a principle can only be a "do this", "thou shalt" sort of statement, outside of time.

Reply:
These are more of reminders? I agree that the burn down work would address this but we need to be diligent about this type of work when we add new code. Are we contributing to the pile of global imports?

Reply:
Yeah fair point, there's nothing wrong with having these as principles until the problem's gone